components/slack/sources/new-message-in-channels/new-message-in-channels.js

```diff
@@ -128,6 +134,9 @@ module.exports = {
     if (this.ignoreMyself && event.user == this.slack.mySlackId()) {
       return
     }
+    if (this.ignoreBot && event.subtype != null && event.subtype == "bot_message") {
```

Comment:
`event.subtype != null` is redundant here.

Reply:
You're right, my bad